src/core/evaluator.js

```diff
@@ -460,6 +461,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
                                               [objId, 'FontType3Res', null]);
         }
         this.handler.send('obj', [objId, this.pageIndex, 'Image', null]);
+        return undefined;
```

Comment:
If we return in both the `then` and `catch` clauses, isn't all code below dead code now then?

Reply:
Sorry, but I really don't follow. This is inside of a `Promise`, hence this only explicitly spells out the return value (as opposed to implicitly previously).

Reply:
Yes, it's fine. I didn't see that this was inside a `Promise`.